Mobile Test Runner for Jama/Mobile Test Runner for Jama/LoginViewController.swift

```diff
@@ -117,6 +118,7 @@ extension LoginViewController: EndpointDelegate{
             //Notify user of unauthorized reload the view on the main thread
             DispatchQueue.main.async {
                 self.reloadViewWithUsernameAndInstanceSaved()
+                self.viewDidLoad()
```

Comment:
This fixed the bug where the Done button doesn't get rid of the keyboard after the error message is displayed.  Is there a better way to reset everything?